demo/api/src/config/config.ts

```diff
@@ -29,6 +29,13 @@ export function createConfig(processEnv: NodeJS.ProcessEnv) {
             ...cometConfig.dam,
             secret: envVars.DAM_SECRET,
         },
+        azure: {
+            translator: {
+                endpoint: envVars.AZURE_AI_TRANSLATOR_ENDPOINT,
+                key: envVars.AZURE_AI_TRANSLATOR_KEY,
+                region: envVars.AZURE_AI_TRANSLATOR_REGION,
+            },
+        },
```

Comment:
You can combine azure and translator to azureAiTranslator. Also, it should be optional